src/document.h

```diff
@@ -745,6 +745,15 @@ struct Document {
         #ifndef __WXMSW__
         if (sw) sw->Refresh(false);
         #endif
+        #ifdef __WXGTK__
+        if (sw) {
+            // wxWidgets (wxGTK) does not always automatically update the scrollbar 

```

Comment:
This calls both the above `Refresh` and your new code GTK, are you sure both are needed?

Reply:
Yes, unfortunately. I tested it with a TreeSheets with nested grids and this is the way I found out to remove the bug that the scrollbar is at default position (0,0) when zoomed in and out.

Reply:
Bug demonstrated:

https://github.com/aardappel/treesheets/assets/3286756/cb34f04c-f553-40b1-8fd0-cc127abaa8fa

Reply:
With  this PR

https://github.com/aardappel/treesheets/assets/3286756/2bbd8496-8ea6-4595-afff-07a6a60eebbe

Reply:
@aardappel Do you agree with the rationale behind this PR?

Reply:
Yes, thanks!